edge-util/src/Microsoft.Azure.Devices.Edge.Util/Compression.cs

```diff
@@ -12,7 +13,7 @@ public static byte[] CompressToGzip(byte[] input)
             {
                 using (var gzipStream = new GZipStream(compressedStream, CompressionMode.Compress))
                 {
-                    gzipStream.Write(input, 0, input.Length);
+                    new MemoryStream(input).CopyTo(gzipStream);
```

Comment:
why use MemoryStream? any benefits?

Reply:
nope, I had originally changed this function to take an ienumerable, but I changed it back. these do the same thing. I didn't realize I'd implemented it differently. I'll revert it back.